docs/source/bounded_variables.rst

```diff
@@ -0,0 +1,54 @@
+=================
+Bounded Variables
+=================
+
+Some variables have constraints.  For instance, it makes no sense for a
+standard deviation to have a negative value, so something like a Normal prior
+on a parameter that represents a standard deviation would be inappropriate.
+PyMC3 includes distributions that have positive valued support, such as
+``Gamma`` or ``Exponential``.  PyMC3 also includes several bounded
+distributions, such as ``Uniform``, ``HalfNormal``, and ``HalfCauchy``, that
+are restricted to a specific domain.  Variables that have been assigned these
+distributions are automatically transformed to remove the discontinuity of the
+probability distribution introduced at zero. 
+
+PyMC3 includes the construct ``Bound`` for placing constraints on existing
```

Comment:
Maybe split what `Bound` is doing and what it can be used for?
"`Bound` allows us to define new distributions that can only take values in a specified interval. For continuous distribution it uses a transformation to allow sampling algorithms to still work on an unconstrained domain."
Also, we should mention that the resulting distribution isn't normalized. I think this can be a problem in eg model comparison, and also if the bounds of the distribution aren't constant.